components/email_aliases/resources/email_aliases_panel.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { createRoot } from 'react-dom/client'
+import * as React from 'react'
+import { StyleSheetManager } from 'styled-components'
+import {
+  EmailAliasModal,
+  EmailAliasModalActionType,
+  EmailAliasModalAction,
+} from './content/email_aliases_modal'
+import BraveCoreThemeProvider from '$web-common/BraveCoreThemeProvider'
+import {
+  AuthenticationStatus,
+  AuthState,
+  Alias,
+  EmailAliasesServiceInterface,
+  EmailAliasesServiceObserverInterface,
+  EmailAliasesServiceObserverReceiver,
+  EmailAliasesService,
+  EmailAliasesPanelHandlerInterface,
+  EmailAliasesPanelHandler,
+} from 'gen/brave/components/email_aliases/email_aliases.mojom.m'
+
+export const EmailAliasesPanelConnected = ({
+  emailAliasesService,
+  emailAliasesPanelHandler,
+  bindObserver,
+}: {
+  emailAliasesService: EmailAliasesServiceInterface
+  emailAliasesPanelHandler: EmailAliasesPanelHandlerInterface
+  bindObserver: (observer: EmailAliasesServiceObserverInterface) => () => void
+}) => {
+  const [authState, setAuthState] = React.useState<AuthState>({
+    status: AuthenticationStatus.kStartup,
+    email: '',
+    errorMessage: undefined,
+  })
+  const [aliasesState, setAliasesState] = React.useState<Alias[]>([])
+  React.useEffect(() => {
+    let status: AuthenticationStatus = AuthenticationStatus.kStartup
+    const observer: EmailAliasesServiceObserverInterface = {
+      onAliasesUpdated: (aliases: Alias[]) => {
+        if (status !== AuthenticationStatus.kAuthenticated) {
+          return
+        }
+        setAliasesState(aliases)
+      },
+      onAuthStateChanged: (state: AuthState) => {
+        status = state.status
+        setAuthState(state)
+        if (status !== AuthenticationStatus.kAuthenticated) {
+          setAliasesState([])
+        }
+      },
+    }
+    return bindObserver(observer)
+  }, [])
+  return (
+    <EmailAliasModal
+      onReturnToMain={(action: EmailAliasModalAction) => {
+        switch (action.type) {
+          case EmailAliasModalActionType.Cancel:
+            emailAliasesPanelHandler.onCancel()
+            break
+          case EmailAliasModalActionType.Manage:
+            emailAliasesPanelHandler.onManage()
+            break
+          case EmailAliasModalActionType.Complete:
+            emailAliasesPanelHandler.onComplete(action.email)
+            break
+        }
+      }}
+      editing={false}
+      mainEmail={authState.email}
+      aliasCount={aliasesState.length}
+      emailAliasesService={emailAliasesService}
+      bubble={true}
+    />
+  )
+}
+
+const mount = () => {
+  const rootElement = document
+    .getElementById('mountPoint')!
+    .attachShadow({ mode: 'open' })
```

Comment:
Hey, I don't think we should need a `shadowRoot` here - the only reason we need it for the other WebUI is because we're mountain the component inside a WebComponent

Probably you don't need to tell the `StyleSheetManager` a `target` either (it should just use `document` by default)